requirements.txt

```diff
@@ -1,4 +1,4 @@
-apache-beam==2.25.0
+apache-beam[azure]==2.27.0
```

Comment:
Just curious, what does `[azure]` do?

Reply:
It adds some Azure Blob Storage dependencies. Functionally, I don't think it makes a difference, since we already have `azure-storage-blob` and `azure-core` in our dependencies. If Beam ends up adding additional dependencies to use Azure (or requires different versions of the Azure client) in the future, though, that would be useful. Additionally, once we support something like Google Cloud Storage, we can just say `apache-beam[azure,gcp]` to pull down all the required dependencies for GCP as well.